test/test.js

```diff
@@ -15,7 +16,7 @@ it('should show the stack', function (done) {
   .expect(500, function (err, res) {
     if (err) return done(err)
     res.body.stack.should.be.ok
-    done()
+    return done()
```

Comment:
Not on the same page here. Also, untrue for a good chunk of cases. It is, in my personal opinion, always (or almost always) good practice to `return` callbacks explicitly. [Not](http://eslint.org/docs/rules/callback-return) [just](http://www.bennadel.com/blog/2323-use-a-return-statement-when-invoking-callbacks-especially-in-a-guard-statement.htm) [my opinion](https://blog.risingstack.com/node-js-best-practices/).